runner_manager/backend/aws.py

```diff
@@ -0,0 +1,109 @@
+        return client("ec2", region_name=self.config.region)
+
+    def create(self, runner: Runner) -> Runner:
+        """Create a runner."""
+        labels: Dict[str, str] = {}
+        if self.manager:
+            labels["runner-manager"] = self.manager
+        self.instance_config.subnet_id = self.config.subnet_id
+        instance_resource: Dict = self.instance_config.configure_instance(runner)
+        try:
+            instance = self.client.run_instances(**instance_resource)
+            runner.instance_id = instance["Instances"][0]["InstanceId"]
+            # Wait for instance to be running
+            waiter = self.client.get_waiter("instance_running")
+            waiter.wait(InstanceIds=[runner.instance_id])
+        except Exception as e:
+            log.error(e)
+            raise e
+        return super().create(runner)
+
+    def delete(self, runner: Runner):
+        """Delete a runner."""
+        if runner.instance_id:
+            try:
+                self.client.terminate_instances(InstanceIds=[runner.instance_id])
+            except ClientError as e:
+                error = e.response.get("Error", {})
+                if error.get("Code") == "InvalidInstanceID.NotFound":
+                    log.error(f"Instance {runner.instance_id} not found.")
+                    return super().delete(runner)
+                elif error.get("Code") == "InvalidInstanceID.Malformed":
+                    log.error(f"Instance {runner.instance_id} malformed.")
+                    return super().delete(runner)
+                else:
+                    raise e
+        return super().delete(runner)
+
+    def list(self) -> List[Runner]:
+        """List runners."""
+        runners: List[Runner] = []
+        try:
+            reservations = self.client.describe_instances(
+                Filters=[
+                    {
+                        "Name": "tag-key",
+                        "Values": ["runner-manager"],
+                    },
+                    {
+                        "Name": "instance-state-name",
+                        "Values": ["running"],
+                    },
+                ]
+            ).get("Reservations")
+        except Exception as e:
+            log.error(e)
+            raise e
+        for reservation in reservations:
+            for instance in reservation.get("Instances", []):
+                instance_id = instance.get("InstanceId", "")
+                try:
+                    runners.append(
+                        Runner.find(
+                            Runner.instance_id == instance_id,
+                        ).first()
+                    )
+                except NotFoundError:
+                    runner = Runner(
+                        name=instance_id,
+                        instance_id=instance_id,
+                        busy=False,
+                    )
+                    runners.append(runner)
+        return runners or []
```

Comment:
```suggestion
        return runners
```
As `runners` is already defined as an empty list, no need to this